pg_search/src/api/index.rs

```diff
@@ -329,42 +338,50 @@ pub fn parse(query_string: String) -> SearchQueryInput {
     SearchQueryInput::Parse { query_string }
 }
 
+#[pg_extern(immutable, parallel_safe)]
+pub fn parse_with_field(field: FieldName, query_string: String) -> SearchQueryInput {
+    SearchQueryInput::ParseWithField {
+        field: field.into_inner(),
+        query_string,
+    }
+}
+
```

Comment:
@rebasedming this is a new query builder function.  To be used in situations like:

```sql
SELECT * FROM t WHERE t @@@ paradedb.parse_with_field('body', 'beer');
```